### ISSUE TITLE
Calibration code

### ISSUE THREAD

**Issue by jmlzone** [jmlzone/PiPtr#4]
What should the code to to help with turn up and calibration.
Ideas here.

**Comment by jmlzone** [comment]
There should be a "find me" mode at start up after loading the xml, the ports though use the audio path from the onboard pwm's to find which usb audio device they are connected to so if they enumerate differences that are properly associated with the correct port.

Also it should be possible to to a soft AGC to correct people with low or loud audio.

**Comment by scivision** [comment]
Yes there are a lot of beautiful things that can be done with the audio, particularly useful for linked systems, especially systems involving internet stations and/or 900 MHz repeaters.

**Comment by jmlzone** [comment]
See portDetect.py (stand alone test) and the portDetect code in hwio.py, this starts to send tones from one sound card port to another for detection and can figure out which sound card is on which radio port.  its not perfect yet, but its a start.
